dask_expr/groupby.py

```diff
@@ -294,6 +294,9 @@ def chunk_kwargs(self):
     def combine_kwargs(self):
         return {"levels": self.levels}
 
+    def _divisions(self):
+        return (None, None)
```

Comment:
Just a small note that this will eventually need to depend on the value of `split_out`.